### PR TITLE
Small fix in HLT b-tag validation code (75X)

### PR DESCRIPTION
**Pull request by silviodonato**
This PR fixes:
- setEfficiencyFlag() option for mistag vs efficiency plots
- trigger name HLT_PFMET120_NoiseCleaned_BTagCSV07_v -> HLT_PFMET120_NoiseCleaned_BTagCSV0p72_v
- trigger name HLT_CaloMHTNoPU90_PFMET90_PFMHT90_IDLoose_ -> HLT_CaloMHTNoPU90_PFMET90_PFMHT90_IDTight_

### CODE REVIEW

**Comment by deguio** [comment]
please test

**Comment by deguio** [comment]
+1

File: HLTriggerOffline/Btag/src/HLTBTagHarvestingAnalyzer.cc [hunk 1]
```diff
@@ -151,7 +151,9 @@ void HLTBTagHarvestingAnalyzer::mistagrate(DQMStore::IBooker& ibooker, DQMStore:
 		eff->SetBinContent(binX,miseff);
 		eff->SetBinError(binX,miseffErr);
 	}
-	ibooker.book1D(effName.c_str(),eff);
+	MonitorElement *me;
+	me = ibooker.book1D(effName.c_str(),eff);
+	me->setEfficiencyFlag();
```
Comment (davidlange6) [inline HLTriggerOffline/Btag/src/HLTBTagHarvestingAnalyzer.cc]: does this need to be deleted?

**Comment by davidlange6** [comment]
+1

**Pull request merged**